src/app/client/src/app/modules/core/services/search/search.service.ts

```diff
@@ -361,8 +363,7 @@ export class SearchService {
       filters['channel'] = request.channelId;
     }
     if (isCourse) {
```

Comment:
instead of **isCourse** pass the contentTypes as an array so that you can replace and easy from component to change and make ['TextBook'] as default